detect/reader.go

```diff
@@ -8,11 +8,11 @@ import (
 )
 
 // DetectReader accepts an io.Reader and a buffer size for the reader in KB
-func (d *Detector) DetectReader(r io.Reader, bufSize int) ([]report.Finding, error) {
-	reader := bufio.NewReader(r)
-	buf := make([]byte, 0, 1000*bufSize)
-	findings := []report.Finding{}
-
+func (d *Detector) DetectReader(r io.Reader) ([]report.Finding, error) {
```

Comment:
I think it's safe to remove `bufSize` here? It looks like the value was only used in `reader_test`, but removing it has no effect.